src/azure-cli/azure/cli/command_modules/iot/commands.py

```diff
@@ -157,9 +157,9 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
 
     with self.command_group('iot central app', iot_central_sdk, client_factory=iot_central_service_factory,
                             is_preview=True) as g:
-        g.custom_command('create', 'iot_central_app_create')
+        g.custom_command('create', 'iot_central_app_create', supports_no_wait=True)
```

Comment:
Is there a wait command to query operation status?

Reply:
I'm not sure, but supports_no_wait is used for turn on wait and no_wait mode

Reply:
Usually we need "wait" command if there is long running operations in this command group because if users use "no_wait" for create/delete, they need another "wait" command to wait for async operations completed.

---
In reply to: [473432713](https://github.com/Azure/azure-cli/pull/14613#discussion_r473432713) [](ancestors = 473432713)